nixos/modules/security/wrappers/wrapper.nix

```diff
@@ -8,6 +8,7 @@ stdenv.mkDerivation {
   hardeningEnable = [ "pie" ];
   CFLAGS = [
     ''-DWRAPPER_DIR="${parentWrapperDir}"''
+    ''-DSOURCE_PROG="${sourceProg}"''
```

Comment:
Now I've looked at the other changes to wrappers around that time by the same fellow, and I can't see why that was done (at first I thought it was a prelude to deduplication, but deduplication wasn't done afterwards). Given the original author's lack of activity in nixos recently, would you consider it churlish to tag him and ask?

I'd like us to do try to investigate the reason (I'll try to spelunk more for it too; pointers are appreciated), but not to argue that an unknown reason must have existed and therefore continues to exist.

Reply:
https://github.com/NixOS/nixpkgs/pull/16654#discussion_r97961889 seems to be arguing for a single wrapper, so that it's more likely that it's cached. @7c6f434c do you still think that the wrapper being same for each suid/sgid/setcap binary is important?

Reply:
BTW. It might not have been obvious, but the "state before cca2e1155617304b5bcb3457309983cf8e99b067" was the state in a middle of a PR (d8ecd5eb0d30c3dc302e336755a4a1d6d0cb1ba4 comes from the same PR #16654, which wasn't squashed).